lib/dal/src/schema/variant/authoring.rs

```diff
@@ -350,10 +348,10 @@ impl VariantAuthoringClient {
             )
             .await?;
 
-        let schema_spec = metadata.to_spec(new_variant_spec)?;
+        let schema_spec = metadata.to_schema_spec(new_variant_spec)?;
         //TODO @stack72 - figure out how we get the current user in this!
         let pkg_spec = PkgSpec::builder()
-            .name(&metadata.name)
+            .name(&metadata.schema_name)
```

Comment:
this is the value from line 332 `name`, right? just trying to follow 😅

Reply:
Yup, it gets packaged into the metadata object